cluster.go

```diff
@@ -144,6 +144,9 @@ type ClusterConfig struct {
 	// (default: 200 microseconds)
 	WriteCoalesceWaitTime time.Duration
 
+	// Dialer
```

Comment:
Maybe the doc comment could be expanded like
```
// Dialer will be used to establish all connections created for this Cluster.
// If not provided, a default dialer configured with ConfigTimeout will be used.
```

Reply:
Thanks! I've added your comment.